queries/futures/useGetAverageFundingRateForMarkets.ts

```diff
@@ -46,8 +49,12 @@ const useGetAverageFundingRateForMarkets = (
 		}
 	);
 
+	const periodLength = PERIOD_IN_SECONDS[period];
+
+	const periodTitle = period === Period.ONE_HOUR ? '1H Funding Rate' : 'Funding Rate';
```

Comment:
And noticed another place where we should probably translate

Reply:
I fixed this, and another for the instantaneous funding rate. That should be all (except the keys, which need to be refactored entirely)